XamarinCommunityToolkit/Views/AvatarView.shared.cs

```diff
@@ -31,6 +31,8 @@ public class AvatarView : BaseTemplatedView<Frame>
 			RGB(255, 255, 255)
 		};
 
+		public static readonly BindableProperty AspectProperty = BindableProperty.Create(nameof(Aspect), typeof(Aspect), typeof(AvatarView), Forms.Aspect.AspectFill, propertyChanged: OnAspectPropertyChanged);
```

Comment:
if we use OnValuePropertyChanged won't it reset Source each time and tries to download image?

Reply:
I think propertyChanged won't be triggered in such cases because the old value and new value are literally the same

https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Core/BindableObject.cs#L426